please.sh

```diff
@@ -345,6 +360,60 @@ copy_to_clipboard() {
   esac
 }
 
+init_questions() {
+  systemPrompt="You will give answers in the context of the command \"${command}\" which is a Linux bash command related to the prompt \"${commandDescription}\". Be precise and succinct, answer in full sentences, no lists, no markdown."
+  escapedPrompt=$(printf %s "${systemPrompt}" | jq -srR '@json')
+
+  qaMessages+=("{ \"role\": \"system\", \"content\": ${escapedPrompt} }")
+}
+
+ask_question() {
+  echo ""
+  # shellcheck disable=SC2059
+  printf "${questionMark} ${cyan}What do you want to know about this command?${black}\n"
+  read -r question
+  answer_question_about_command
+
+  echo "${answer}"
+
+  # shellcheck disable=SC2059
+  printf "${checkMark} ${answer}\n"
+
+  choose_action
+  act_on_action
+}
+
+answer_question_about_command() {
+  prompt="${question}"
```

Comment:
NIT: I think it would be good to use `local`, for variables that are just used inside the scope of the function:
```suggestion
  local prompt="${question}"
```

Reply:
I think we should do the local variable declaration for the whole file, and probably not as part of this PR.